controller/types/types.go

```diff
@@ -349,3 +350,20 @@ type DomainMigrationEvent struct {
 	DomainMigration *DomainMigration `json:"domain_migration"`
 	Error           string           `json:"error,omitempty"`
 }
+
+const (
+	ClusterBackupStatusRunning  string = "running"
+	ClusterBackupStatusComplete string = "complete"
+	ClusterBackupStatusError    string = "error"
+)
+
+type ClusterBackup struct {
+	ID          string     `json:"id,omitempty"`
+	Status      string     `json:"status"`
+	SHA512      string     `json:"sha512,omitempty"`
+	Size        int32      `json:"size,omitempty"`
```

Comment:
Something in the db stack was complaining about it being an int during deserialization.